presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -819,6 +840,12 @@ public void testArrayAsJson()
                 .addRoundTrip(arrayAsJsonDataType("_text"), "[]")
                 .execute(getQueryRunner(), session, postgresCreateAndInsert("tpch.test_varchar_array_as_json"));
 
+        testUnsupportedDataTypeAsIgnored(session, "bytea[]", "ARRAY['binary value'::bytea]");
+        testUnsupportedDataTypeAsIgnored(session, "bytea[]", "ARRAY[ARRAY['binary value'::bytea]]");
+        testUnsupportedDataTypeAsIgnored(session, "bytea[]", "ARRAY[ARRAY[ARRAY['binary value'::bytea]]]");
+        testUnsupportedDataTypeAsIgnored(session, "_bytea", "ARRAY['binary value'::bytea]");
+        testUnsupportedDataTypeConvertedToVarchar(session, "bytea[]", "ARRAY['binary value'::bytea]", "'{\"\\\\x62696e6172792076616c7565\"}'");
```

Comment:
testUnsupportedDataTypeConvertedToVarchar and testUnsupportedDataTypeAsIgnored creates a table per each test. Hence it could take a long time to test all use cases. We had this issue before with regular type mapping tests. Maybe we should think to extend that framework to support unsupported type testing.
WDYT?
No change requested, just thinking at loud.

Reply:
I added a few cases because we they apparently differ in execution flow within PostgreSQL JDBC driver.
`testArrayAsJson` took 3.2s and `testArray` took 9.8s for me.

> Maybe we should think to extend that framework to support unsupported type testing.

Let's defer this until we have more of these.